common/djangoapps/util/views.py

```diff
@@ -150,8 +155,39 @@ def update_ticket(self, ticket_id, update):
         """
         self._zendesk_instance.update_ticket(ticket_id=ticket_id, data=update)
 
+    def get_group(self, name):
```

Comment:
Note for reviewers: I went with this approach over adding the Financial Assistance group ID as a setting, since that seems quite brittle to changes in our Zendesk settings.